#122, 115

vitaly-savin opened the pull request:
Created theHire us! page, customer form, added validation. Fixed some ESLint warnings

vitaly-savin committed:
#122, 115

Created theHire us! page, customer form, added validation. Fixed some ESLint warnings

leresolu commented on utils/myUtils.js:
Please use UUID npm package instead: https://www.npmjs.com/package/uuid

vitaly-savin replied:
done

leresolu commented:
Are those changes not related to story #70? Why is it being merged in #102?

leresolu commented on pages/Hire.vue:
```diff
@@ -1,54 +1,120 @@
+    <v-container
+        class="pa-0"
+        fluid
+        style="
```
pleae remove custom styles leftover from wordpress and use vuetify only

vitaly-savin replied:
done

leresolu commented on pages/Hire.vue:
```diff
@@ -1,54 +1,120 @@
+            background-image: linear-gradient(180deg,rgba(9,9,21,0) 0%,#090915 100%), url(./img/bcground-home.jpg)!important;">
+        <span
+            style="
+            position: absolute;
```
same here

vitaly-savin replied:
done

leresolu commented on pages/Hire.vue:
```diff
@@ -1,54 +1,120 @@
+                        <v-btn 
+                            elevation="2"
+                            href="/members"
+                            style="background-color: #FFBB01; color:rgb(224, 43, 32); font-size:20px; text-shadow: 0.08em 0.08em 0.08em rgba(0,0,0,0.4); letter-spacing:4px; font-weight:700; padding: 24px 20px;">
```
same here

vitaly-savin replied:
done

leresolu commented on pages/Hire.vue:
```diff
@@ -1,54 +1,120 @@
-            }
+    methods: {
+        customerCreated(newCustomer) {
+            const newId = generateUUID(5)
```
use UUID npm package instead

vitaly-savin replied:
done

vitaly-savin committed:
Fixed requested changes

1) removed custom styles leftover from wordpress and use vuetify only
2) use UUID npm package
